### ISSUE TITLE
is there branch build by maven?

### ISSUE THREAD

**Issue by TRUEJASONFANS** [SilverTiger/lwjgl3-tutorial#14]
just like the titile

**Comment by SilverTiger** [comment]
No, but it should be easy to migrate to Maven. The dependencies and version is in the `ivy.xml` and in the `ivysettings.xml`. The fine tuning of the building can be found in the `build.xml`.
